app/assets/stylesheets/application.scss

```diff
@@ -16,6 +16,7 @@ $hmrc-assets-path: "/report-quarterly/income-and-expenses/view/assets/lib/hmrc-f
 
 .hmrc-account-menu {
   border-top: 1px solid #b1b4b6;
+  font-size: 1rem
```

Comment:
I think this needs to go into .hmrc-account-menu__link instead

Reply:
Will fix 👍